manager/gui/tests/unit/LrSystemHealthConfiguration.spec.ts

```diff
@@ -0,0 +1,181 @@
+import { mount, createLocalVue } from "@vue/test-utils";
```

Comment:
scaffold the same folder structure like alerts, configuration so that spec files are also segregated module wise.

Reply:
Done